README.md

```diff
@@ -47,19 +47,25 @@ More detailed build instructions can be found [here](docs/install_vortex.md).
 - [Yosys](https://github.com/YosysHQ/yosys)
 - [Sv2v](https://github.com/zachjs/sv2v)
 ### Install development tools
-    $ sudo apt-get install build-essential
-    $ sudo apt-get install binutils
-    $ sudo apt-get install python
-    $ sudo apt-get install uuid-dev
-    $ sudo apt-get install git
+    $ sudo apt-get install build-essential git binutils python3 uuid-dev wget ocl-icd-opencl-dev
 ### Install Vortex codebase
-    $ git clone --depth=1 --recursive https://github.com/vortexgpgpu/vortex.git
-    $ cd Vortex
+    $ git clone https://github.com/vortexgpgpu/vortex.git
+    $ cd vortex
+    $ git submodule sync
+    $ git submodule update --init --recursive --jobs 6
 ### Configure your build folder
     # By default, the toolchain default install location is the /opt folder and can be overridden by setting --tooldir.
     $ mkdir build
     $ cd build
     $ ../configure --xlen=32 --tooldir=$HOME/tools
+
+### Only Ubuntu Jammy(22.04) and Mantic (23.04) fix
+    ensure that 
+    #include <cstdint>
+    is added to  ./third_party/ramulator/src/StatType.h
+    the configure script should have done this but just make sure as this is a
+    subrepo
+
```

Comment:
If it turns out the source code modifications are necessary
then I'm not fully convinced that overwriting it here is the best approach.

Did you check if ramulator has a new version that we might be able to upgrade to that resolves the issues you encountered?

Reply:
I have not checked this but I could look into this. yeah I dont like this approach its pretty hacky

I am using ubuntu 23 mantic.

Reply:
![image](https://github.com/user-attachments/assets/0c6e54fd-a3ba-4f2c-8f8e-3a68a7b80a35)